dynamicio/config.py

```diff
@@ -104,6 +104,53 @@ def dyn_str_constructor(self, node: yaml.nodes.ScalarNode) -> str:
         return value
 
 
+class SafeDynamicSchemaLoader(yaml.SafeLoader):
+    """Implements a dynamic yaml loader that parses yaml files and replaces strings that map to [[ DYNAMIC_VAR ]].
+
+    Dynamic variables defined in a provided module object.
+    """
+
+    module = None
+    dynamic_data_matcher = re.compile(r"(.*)(\[\[\s*(\S+)\s*]])(.*)")
```

Comment:
I was surprised that we're matching also on anything before and after the  '[[ ' and ' ]]', may I ask why this is the case?

Reply:
What happens if you want to use something like: `"[[ VALUE_1 ]]/[[ VALUE_2 ]]"'`